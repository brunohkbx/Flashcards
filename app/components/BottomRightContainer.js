import { View } from 'react-native';
import styled from 'styled-components';

const BottomRightContainer = styled(View)`
  position: absolute;
  right: ${props => props.right};
  bottom: ${props => props.right}; 
`;

export default BottomRightContainer;
