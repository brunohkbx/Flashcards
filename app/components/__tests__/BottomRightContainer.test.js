import React from 'react';
import { shallow } from 'enzyme';
import BottomRightContainer from '../BottomRightContainer';

describe('BottomRightContainer', () => {
  const setup = propOverrides => {
    const defaultProps = Object.assign({
      right: 16,
      bottom: 16
    }, propOverrides);

    const wrapper = shallow(<BottomRightContainer {...defaultProps} />);

    return { wrapper };
  };

  it('renders properly', () => {
    const { wrapper } = setup();

    expect(wrapper).toMatchSnapshot();
  });
});
