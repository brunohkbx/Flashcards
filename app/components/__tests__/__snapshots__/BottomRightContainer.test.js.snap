// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomRightContainer renders properly 1`] = `
<Component
  bottom={16}
  className="sc-bdVaJa fkjgPF"
  right={16}
/>
`;
